rotkehlchen/data_import/importers/binance.py

```diff
@@ -98,6 +98,63 @@ def process_entries(
         """Turns given csv rows into internal rotki's representation"""
 
 
+class BinanceTransferEntry(BinanceMultipleEntry):
+    def are_entries(self, requested_operations: list[str]) -> bool:
+        """
+        This class supports several formats of Transfers from the csv.
+        Supports the following combinations of "Operation" column's values:
+            - Transfer Between Spot Account and UM Futures Account
+            - Transfer Between Spot Account and CM Futures Account
+            - Transfer Between Main and Funding Wallet
+        """
+        return len(requested_operations) > 1 and len(requested_operations) % 2 == 0 and requested_operations[0].startswith('Transfer Between ')  # noqa: E501
+
+    @staticmethod
+    def process_transfers(
+            timestamp: Timestamp,
+            data: list[BinanceCsvRow],
+    ) -> list[HistoryBaseEntry]:
+        """
+        Processes transfer events
+        May raise:
+        - KeyError
+        """
+        event_identifier = f'{EVENT_IDENTIFIER_PREFIX}{hash_csv_row(data[0])}'
+        timestamp_ms = ts_sec_to_ms(timestamp)
+        transfer_events: list[HistoryBaseEntry] = [
+            HistoryEvent(
+                event_identifier=event_identifier,
+                sequence_index=index,
+                timestamp=timestamp_ms,
+                location=Location.BINANCE,
+                event_type=HistoryEventType.TRANSFER,
+                event_subtype=HistoryEventSubType.NONE,
+                asset=row['Coin'],
+                balance=Balance(amount=abs(row['Change'])),
```

Comment:
Hmm you are performing `abs()` on a raw string/int value from the csv. This should be deserialized as an FVal properly and checked for errors. Is it? I guess it's done in `_group_binance_rows`. Same as for the `asset` in `csv_row['Coin']`.

But that is very hard to understand and error prone. Is it possible to change the csv row to a typed dict? We need to be sure of the types flying around in these functions. Otherwise I see a `row['key']` and I assume, and so will every new reader, that this is the raw string/int from the CSV.